reporter/build.gradle.kts

```diff
@@ -60,6 +62,36 @@ repositories {
             includeGroup("bad.robot")
         }
     }
+
+    exclusiveContent {
+        forRepository {
+            maven("https://download.eclipse.org/antenna/releases/")
+        }
+
+        filter {
+            includeGroup("org.eclipse.sw360.antenna")
+        }
+    }
+
+    exclusiveContent {
+        forRepository {
+            maven("https://jitpack.io")
+        }
+
+        filter {
+            includeGroup("com.github.ralfstuckert.pdfbox-layout")
+        }
+    }
+}
+
+configurations.all {
+    resolutionStrategy.dependencySubstitution {
+        substitute(module("com.github.oss-review-toolkit.ort:spdx-utils")).apply {
+            with(project(":spdx-utils"))
+            because("attribution-document-core depends on ORT's spdx-utils artifact from JitPack, but we want " +
+                    "to use the belonging unreleased project for development")
+        }
+    }
```

Comment:
> I guess that long configuration block is the reason why you wanted to get rid of the reporter dependency of helper-cli?

Correct.

> Is there no way to avoid repeating it?

Well, we maybe could move it to a function in `buildSrc` or so.

> If another project would depend on our reporter module from jitpack, would it have to add the same configuration?

Not the dependency substitution part, no. The dependency substitution only makes sense (and is possible) as we have the source code of a dependency that is pulled in as a binary artifact.